### DISCUSSION TITLE
msquic and .NET SDK version correspondence

### DISCUSSION THREAD

**Discussion by alexandrehtrb** [microsoft/msquic#3911] [General]
Hello,

Is there a corresponding version of .NET SDK / runtime version to msquic version?
If yes, where is it available?

**Comment by alexandrehtrb** [comment]
Is it defined here?

https://github.com/dotnet/runtime/blob/fb39575ad01c64f276fc1312a42189539828ffba/eng/Versions.props#L182

**Reply by ManickaP** [reply]
This one defines the exact version of MsQuic package we bundle together with .NET on Windows. In general, we specify minimal MsQuic version here:
https://github.com/dotnet/runtime/blob/e8b8a77b0c1364e90263bc565d0c0a5552f53dba/src/libraries/System.Net.Quic/src/System/Net/Quic/Internal/MsQuicApi.cs#L21

Anything higher or equal than that should work.

Also note that we will disable QUIC support on .NET level if we cannot find and load MsQuic lib or if it's not high enough version. So you shouldn't encounter any crashes caused by MsQuic lib version mismatch.

**Reply by alexandrehtrb** [reply]
Thank you
